### PR TITLE
vulkan-headers, vulkan-loader: add ports

### PR DESCRIPTION
**Pull request by ElectrodeYT**

### CODE REVIEW

**Review by no92 — APPROVED** [review]
LGTM

File: bootstrap.d/media-libs.yml [hunk 1]
```diff
@@ -876,6 +876,49 @@ packages:
...
+      tag: 'v1.3.268'
+      version: '1.3.268'
+    tools_required:
+      - system-gcc
+      - host-cmake
+    pkgs_required:
+      - mlibc
+      - libx11
+      # This is only required for building.
+      - vulkan-headers
```
Comment (Dennisbonke) [inline bootstrap.d/media-libs.yml:900]: Potentially missing dependency on wayland, wayland-protocols and libXrandr?

File: bootstrap.d/media-libs.yml [hunk 2]
```diff
@@ -876,6 +876,49 @@ packages:
...
+      maintainer: "Alexander Richards <electrodeyt@gmail.com>"
+      categories: ['media-libs']
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/KhronosGroup/Vulkan-Loader.git'
+      tag: 'v1.3.268'
+      version: '1.3.268'
+    tools_required:
+      - system-gcc
+      - host-cmake
```
Comment (Dennisbonke) [inline bootstrap.d/media-libs.yml:895]: Potentially missing dependency on wayland-scanner?

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Please verify the dependency issues listed, gentoo lists them and if wayland is used then wayland-scanner might well be needed too.

File: bootstrap.d/media-libs.yml (hunk 2 above)
Comment (ElectrodeYT) [reply bootstrap.d/media-libs.yml:895]: AFAICT wayland-scanner is not needed here.

File: bootstrap.d/media-libs.yml (hunk 1 above)
Comment (ElectrodeYT) [reply bootstrap.d/media-libs.yml:903]: Done.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**